src/EventProjectorServiceProvider.php

```diff
@@ -57,7 +57,23 @@ public function register()
         $this->app->singleton(EventProjectionist::class, function () {
             $config = config('event-projector');
 
-            return new EventProjectionist($config);
+            $projectionist = new EventProjectionist($config);
+
+            $projectionist->addProjectors(
+                collect(config('event-projector.projectors', []))
+                    ->map(function ($class) {
+                        return $this->determineClass($class);
```

Comment:
You don't need to instantiate these here. That will already be done inside the projectionist.

Reply:
Right, that makes things more easy 👍